sentinel/src/schedule/replications.js

```diff
@@ -121,7 +121,7 @@ module.exports = {
     return Promise.resolve();
   },
   runReplication: () => {
-    const SRC_DB_REGEX = new RegExp(`${db.medicDbName}-user-[^\\-]+-meta`);
+    const SRC_DB_REGEX = new RegExp(`${db.medicDbName}-user-[a-z0-9_-]+-meta`);
```

Comment:
I think this is going to miss other potentially valid user DBs. In API we have [a regex](https://github.com/medic/cht-core/blob/master/api/src/services/user-db.js#L12) to detect and escape characters in user names that are not valid in DB names so we have to accept at least all those characters.

For example, this perfectly valid DB name fails this regex: `medic-user-stupid+name-meta`.

In fact I think we can be very lenient here - as long as the DB name starts with `${db.medicDbName}-user-` and ends with `-meta` we should be good, so what about just...

```suggestion
    const SRC_DB_REGEX = new RegExp(`${db.medicDbName}-user-.+-meta`);
```

Reply:
Actually this regex `[a-z0-9_-]` is the same used to validate users when created: https://github.com/medic/cht-core/blob/df35966a8a815063e294869b3d6858ee413498a3/api/src/services/users.js#L18

If you try to create a user like the one in your example you will get an error like:

![cht-error-username](https://user-images.githubusercontent.com/1608415/91609309-84eb7b00-e94d-11ea-9f96-6362a7bc8ef1.png)

Are you sure you want to open the regex to forbidden characters?

Reply:
Yes, make the one in the replication code more tolerant. It's possible users were created either before the check was enforced in the UI, or directly via CURL or medic-conf or fauxton.